Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -60,14 +58,34 @@ static inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const C
 };
 
 struct __CGPath : CoreFoundation::CppBase<__CGPath, __CGPathImpl> {
+    inline ComPtr<ID2D1PathGeometry>& PathGeometry() {
+        return _impl.pathGeometry;
+    }
+
+    inline ComPtr<ID2D1GeometrySink>& GeometrySink() {
+        return _impl.geometrySink;
+    }
+
+    inline bool& IsFigureClosed() {
+        return _impl.isFigureClosed;
```

Comment:
I really dislike returning a reference here.  Traditional setters/getters would be more appropriate